src/tls/openssl/tls.c

```diff
@@ -143,7 +154,7 @@ static int keytype2int(enum tls_keytype type)
 
 #if OPENSSL_VERSION_NUMBER >= 0x10100000L && \
 	!defined(LIBRESSL_VERSION_NUMBER)
-static int verify_handler(int ok, X509_STORE_CTX *ctx)
+int tls_verify_handler(int ok, X509_STORE_CTX *ctx)
```

Comment:
I would prefer handlers to be static.

Reply:
Okay.

The perfect name for a function in `tls.c` that turns on SSL_VERIFY_PEER for a TLS client and sets a verify handler would be: `void tls_set_verify_client(tls)`

But there is already a function with this name that does the opposite. It's sets the handler `verify_trust_all`. :-(